opensrp-anc/src/main/java/org/smartregister/anc/library/listener/ContactTaskDisplayClickListener.java

```diff
@@ -0,0 +1,35 @@
+package org.smartregister.anc.library.listener;
+
+import android.app.AlertDialog;
+import android.content.Context;
+import android.view.View;
+
+import org.smartregister.anc.library.R;
+
+public class ContactTaskDisplayClickListener implements View.OnClickListener {
+    @Override
+    public void onClick(View view) {
+        if (view != null) {
+            if (view.getId() == R.id.accordion_info_icon) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [These nested if statements could be combined](https://app.codacy.com/manual/OpenSRP/opensrp-client-anc/pullRequest?prid=4809639)